rpc/swagger/swagger.yaml

```diff
@@ -2052,21 +2066,33 @@ components:
                         - "time_iota_ms"
                       properties:
                         max_bytes:
-                          type: "string"
+                          type: string
                           example: "22020096"
                         max_gas:
-                          type: "string"
+                          type: string
                           example: "1000"
                         time_iota_ms:
-                          type: "string"
+                          type: string
                           example: "1000"
                     evidence:
                       type: "object"
                       required:
-                        - "max_age"
+                        - "max_age_num_blocks"
+                        - "max_age_duration"
+                        - "max_num"
+                        - "proof_trial_period"
                       properties:
                         max_age:
-                          type: "string"
+                          type: string
+                          example: "100000"
+                        max_age_duration:
+                          type: string
+                          example: "100000"
+                        max_num:
+                          type: integer
+                          example: 50
+                        proof_trial_period:
+                          type: string
```

Comment:
you could also add the defaults here if you wanted - don't know if it is necessary though

Reply:
The example is shown an in a JSON object example so default would not make a difference here